pkg/image/apis/image/helper.go

```diff
@@ -355,6 +350,11 @@ func JoinImageStreamTag(name, tag string) string {
 	return fmt.Sprintf("%s:%s", name, tag)
 }
 
+// JoinImageStreamImage creates a name for image stream image object from an image stream name and an id.
+func JoinImageStreamImage(name, id string) string {
```

Comment:
Why the change?  `Make` reads better to me for this purpose.

Reply:
I wanted to make it consistent with the [other helper](https://github.com/openshift/origin/blob/381a1436e1988e9419ee58d4282d1e84d479ec34/pkg/image/apis/image/helper.go#L351) `JoinImageStreamTag`.

Reply:
I'll split it to a separate commit.